### PR TITLE
Add technical lead role description

### PR DESCRIPTION
**Pull request by saschagrunert**
This commit adds a role description for technical leads in the
Kubernetes community.

**Which issue(s) this PR fixes**:

None

/cc @mbbroberg @justaugustus @alejandrox1 @LappleApple @parispittman

### CODE REVIEW

File: contributors/chairs-and-techleads/README.md
Comment (saschagrunert) [inline contributors/chairs-and-techleads/README.md:76]: Not sure what we want to do with this section.

**Comment by nikhita** [comment]
Going to add a hold since this will need steering review
cc @kubernetes/steering-committee 

/hold
/committee steering

**Comment by mbbroberg** [comment]
Hey @saschagrunert -- I was thinking this is the right folder, but would be better as a discrete document since it (rather well!) explains the TL role only. This README needs more improvements and would benefit from noting its contents, including this new file. 

Could you move this info to a `teachnical-lead.md` file, and we can revise the README in another PR?

**Comment by saschagrunert** [comment]
> Could you move this info to a `teachnical-lead.md` file, and we can revise the README in another PR?

Sure, changed as suggested :+1:

**Comment by mbbroberg** [comment]
@saschagrunert from a readership point of view, this is clear, concise, and helps a reader know how to be and become a TL. 

I'm following @nikhita's lead that steering needs to give a thumbs up or down on it. CC @parispittman so you know I helped shepherd from the Slack chat you tagged our team in. 

/approve 
/lgtm

**Review by mbbroberg — APPROVED** [review]

**Comment by saschagrunert** [comment]
Hey, any update about this one? Can I do anything to get this merged? :innocent:

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (dims) [inline contributors/chairs-and-techleads/teachnical-lead.md:73]: I think we can drop this sentence. 73-74.

Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md:73]: Yep, removed that one :+1:

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (nikhita) [reply contributors/chairs-and-techleads/teachnical-lead.md:56]: re: debate - to make it more explicit for someone new reading this maybe we could also include something like "the ability to ask the right questions and say no"?

**Review by nikhita — COMMENTED** [review]
@saschagrunert This is great, thank you so much for writing this down!!

Left some suggestions but I'm also ok with moving them to a follow-up if that's easier. :)

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md]: Yep, added the points to the sentence. :+1:

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md:54]: Yes, added this information as suggested. :+1:

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md:21]: I think every SIG chairs have their own set of necessary permissions, right? Not sure if we more it more explicit.

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md:32]: Hm, I can follow-up on that. :)

**Comment by nikhita** [comment]
Lgtm, we can always iterate if needed :)

/cc @kubernetes/steering-committee 
PTAL

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (liggitt) [inline contributors/chairs-and-techleads/teachnical-lead.md:16]: suggest using one of these URLs that will link to the latest content:

* absolute via redirect: http://git.k8s.io/community/committee-steering/governance/sig-governance.md#roles 
* relative: /committee-steering/governance/sig-governance.md#roles

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (liggitt) [inline contributors/chairs-and-techleads/teachnical-lead.md:50]: nit: find/replace curly quotes everywhere:

```suggestion
Technical leadership within a SIG should focus on solving the "How" questions
```

**Review by liggitt — COMMENTED** [review]
lgtm, just a few nits

**Comment by dims** [comment]
@saschagrunert let's fix the last few nits and ship it. thanks for doing this!

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md]: Fixed by using the absolute URL :+1:

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md]: Ah, good catch. Fixed.

File: contributors/chairs-and-techleads/teachnical-lead.md
Comment (saschagrunert) [reply contributors/chairs-and-techleads/teachnical-lead.md]: Changed as suggested.

**Comment by saschagrunert** [comment]
> @saschagrunert let's fix the last few nits and ship it. thanks for doing this!

Thanks for the hint, fixed the last review round as suggested.

**Comment by mrbobbytables** [comment]
/lgtm

As a follow-up (not needed in this PR), it might be worth linking the tech-lead description from [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#tech-lead) to this doc.

**Comment by dims** [comment]
/hold cancel

we have 3 steering members lgtm-ing :)

**Comment by justaugustus** [comment]
Great work, @saschagrunert!!

**Pull request merged**
